docs/ops/detection/ExperimentalDetectronDetectionOutput_6.md

```diff
@@ -0,0 +1,152 @@
+
+**Short description**: An operation *ExperimentalDetectronDetectionOutput* performs non-maximum suppression to generate the detection output using information on location and score predictions.
+
+**Detailed description**: Operation doing next steps:
+
+1.  Applies deltas to boxes sizes;
+2.  If *class_agnostic_box_regression* is `True` then operation removes predictions for background classes;
+3.  Clips boxes to image;
+4.  Applies *score_threshold* on detection scores;
+5.  Applies non-maximum suppression class-wise with *nms_threshold*;
+6.  If total number of detections then operation leaves only *max_detections_per_image* detections.
+
+**Attributes**:
+
+* *score_threshold*
+
+    * **Description**: *score_threshold* attribute specifies threshold to consider only detections whose score are larger than a threshold.
+    * **Range of values**: non-negative floating point number
+    * **Type**: float
+    * **Default value**: 0.05
+    * **Required**: *yes*
+
+* *nms_threshold*
+
+    * **Description**: *nms_threshold* attribute specifies threshold to be used in the NMS stage.
+    * **Range of values**: non-negative floating point number
+    * **Type**: float
+    * **Default value**: 0.5
+    * **Required**: *yes*
+
+* *num_classes*
+
+    * **Description**: *num_classes* attribute specifies number of detected classes.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: 81
+    * **Required**: *yes*
+
+* *post_nms_count*
+
+    * **Description**: *sampling_ratio* attribute specifies the maximal number of detections per class.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: 2000
+    * **Required**: *yes*
+
+* *max_detections_per_image*
+
+    * **Description**: *max_detections_per_image* attribute specifies maximal number of detections per image.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: 100
+    * **Required**: *yes*
+
+* *class_agnostic_box_regression*
+
+    * **Description**: *class_agnostic_box_regression* attribute ia a flag specifies whether to delete background classes or not.
+    * **Range of values**:
+      * `true` means background classes should be deleted
+      * `false` means background classes shouldn't be deleted
+    * **Type**: boolean
+    * **Default value**: false
+    * **Required**: *yes*
+
+* *max_delta_log_wh*
+
+    * **Description**: *max_delta_log_wh* attribute specifies maximal delta of logarithms for width and height.
+    * **Range of values**: floating point number
+    * **Type**: float
+    * **Default value**: log(1000.0f / 16.0f)
+    * **Required**: *yes*
+
+* *deltas_weights*
+
+    * **Description**: *deltas_weights* attribute specifies deltas of weights.
+    * **Range of values**: a list of non-negative floating point numbers
+    * **Type**: float[]
+    * **Default value**: `[10., 10., 5., 5.]`
+    * **Required**: *yes*
```

Comment:
If this attribute is required, the we needn't specify the default value.

Reply:
Sure, deleted.